src/t_list.c

```diff
@@ -462,14 +462,19 @@ void addListRangeReply(client *c, robj *o, long start, long end, int reverse) {
     }
 }
 
-/* A housekeeping helper for list elements popping tasks. */
-void listElementsRemoved(client *c, robj *key, int where, robj *o, long count) {
+/* A housekeeping helper for list elements popping tasks.
+ * The argument 'deleted' indicates whether the list is deleted. */
```

Comment:
let's clearly indicate it's an "optional, output" argument. and do that in the other functions it's given.